HW_1/task04.py

```diff
@@ -0,0 +1,13 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. 
+# Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, если известно, 
+# что Петя и Сережа сделали одинаковое количество журавликов, 
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+quantity = int(input('Введите количество журавликов: '))
+petaANDsereja = (quantity // 2) // 3
```

Comment:
Во первых, я говорил на семинаре, что тут необходимо проверять входные данные на корректность. Иначе мы можем получить нерелевантные результаты, которые не будут соотноситься с этими входными данными. Во-вторых, у операций деления одинаковый приоритет, то есть скобки не нужны.
```suggestion
petaANDsereja = quantity // 2 // 3
```
В третьих,  в Python используются змеиный шрифт, взамен верблюжьего. И в четвертых, не принято использовать в качестве имен переменных транслитерацию кириллических словесных конструкций.